### PR TITLE
[13.0][FIX] website_sale_product_minimal_price: Fix tests

### PR DESCRIPTION
**Pull request by joao-p-marques**
Ensure we use a consistent currency

@Tecnativa
TT28906

ping @CarlosRoca13 @pedrobaeza

### CODE REVIEW

File: website_sale_product_minimal_price/tests/test_website_sale_product_minimal_price.py [hunk 1]
```diff
@@ -6,13 +6,14 @@
 class WebsiteSaleProductMinimalPriceHttpCase(HttpCase):
     def setUp(self):
         super().setUp()
-
+        # Always work with dollars
+        self.company = self.env.ref("base.main_company")
+        self.company.currency_id = self.env.ref("base.USD")
```
Comment (chienandalu) [inline website_sale_product_minimal_price/tests/test_website_sale_product_minimal_price.py]: Isn't this going to clash with the currency change constrains?

Comment (pedrobaeza) [reply website_sale_product_minimal_price/tests/test_website_sale_product_minimal_price.py]: In fact this is forbidden in code under certain conditions, so we will have another problem in the future.

Comment (joao-p-marques) [reply website_sale_product_minimal_price/tests/test_website_sale_product_minimal_price.py]: Ok, but then what is the best solution? AFAICS the problem is that the test creates a product and assigns a price (not specifying currency) but, when running the tour, the price displayed in the shop follows the default currency set. 
With only this module installed, everything is computed in dollars (default) but, when another module is installed that changes the default currency for the company, it ends up setting a price in euros, but displaying it in dollars (with different values) thus making the tour fail...

Comment (pedrobaeza) [reply website_sale_product_minimal_price/tests/test_website_sale_product_minimal_price.py]: Create a pricelist with that currency and assign it.

**Comment by joao-p-marques** [comment]
Changes made @chienandalu @pedrobaeza 
I think it is correct now, thanks!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**